src/Actions/ResolveForPropertyValueAction.php

```diff
@@ -34,6 +34,6 @@ protected function isValueAnEnum($value): bool
 
         $enumNamespace = is_object($value) ? get_class($value) : $value;
 
-        return ! is_array($value) && enum_exists($enumNamespace);
+        return ! is_array($value) && enum_exists((string) $enumNamespace);
```

Comment:
Doing an explicit string check should be better. As it could also be a boolean or whatever else we have.
```suggestion
        return ! is_array($value) && is_string($enumNamespace) && enum_exists($enumNamespace);
```